test-resources/kt/feature_flag_system_1/control/expected/XPFlagCleanerCases.kt

```diff
@@ -57,7 +57,7 @@ internal class XPFlagCleanerPositiveCases {
         tBool = false
         tBool = false
 
-        tBool = false
+        tBool = true
```

Comment:
nice. This was a bug i guess. 
`tBool = experimentation.isToggleEnabled(TestExperimentName.STALE_FLAG) || true` was evaluating to false earlier :O